framework/bin/run_coverage.pl

```diff
@@ -273,7 +280,12 @@ sub _run_coverage {
 
     my $src_dir = $project->src_dir($vid);
     my $test_log = "$TMP_DIR/.coverage.log"; `>$test_log`;
-    my $cov_info = Coverage::coverage_ext($project, "$TARGET_CLASSES_DIR/$bid.src", $src_dir, $test_dir, $INCL, $test_log);
+    my $cov_info;
+    if (defined $INSTRUMENT_CLASSES) {
```

Comment:
See comment above about desirable behavior if both flags (-i and -A) are provided.